keyboards/cannonkeys/malicious_ergo/info.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
![image](https://user-images.githubusercontent.com/4781841/168956161-f9a6097d-f1a3-4c5a-8c4a-bf854b674da2.png)

Reply:
Oh man :( These ergo layout boards are tough.

I think I used the rotated version to generate the info JSON. Should I not?

Reply:
Nope, rotation isn't supported.

Reply:
This is still outstanding.